x-pack/platform/plugins/shared/lens/public/datasources/form_based/dimension_panel/dimension_editor.tsx

```diff
@@ -421,6 +412,7 @@ export function DimensionEditor(props: DimensionEditorProps) {
       compatibleWithSampling:
         getSamplingValue(state.layers[layerId]) === 1 ||
         (definition.getUnsupportedSettings?.()?.sampling ?? true),
+      quickFunctionDocumentation: definition.quickFunctionDocumentation
```

Comment:
nit pick: I think we could reduce the name here

```suggestion
      documentation: definition.quickFunctionDocumentation
```